docs/fundamentals/code-analysis/quality-rules/ca1870.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "CA1870: Use a cached 'SearchValues' instance"
+description: "Learn about code analyzer rule CA1870 - Use a cached 'SearchValues' instance"
+ms.date: 09/12/2023
+ms.topic: reference
+f1_keywords:
+ - CA1870
+ - UseSearchValuesAnalyzer
+helpviewer_keywords:
+ - CA1870
+dev_langs:
+ - CSharp
+author: MihaZupan
+ms.author: mizupan
+---
+
+# CA1870: Use a cached 'SearchValues' instance
+
+| Property                            | Value                                  |
+|-------------------------------------|----------------------------------------|
+| **Rule ID**                         | CA1870                                 |
+| **Title**                           | Use a cached 'SearchValues' instance   |
+| **Category**                        | [Performance](performance-warnings.md) |
+| **Fix is breaking or non-breaking** | Non-breaking                           |
+| **Enabled by default in .NET 7**    | No                                     |
```

Comment:
I copied this from existing CA docs, does it need changing?

This analyzer is being added in .NET 8, where it should be enabled by default.

Reply:
Yes, let's update it to .NET 8.